experimental/dds/tree2/src/core/tree/cursor.ts

```diff
@@ -363,6 +363,36 @@ export function forEachNode<TCursor extends ITreeCursor = ITreeCursor>(
 	}
 }
 
+/**
+ * Moves cursor to each child of the current field.  The optional arguments `start` and `end`
```

Comment:
I generally prefer one sentence per line. It aligns our line based tooling (like git, diffs, code review tools etc) with our semantics better than line wrapping in the middle of sentences.

It also avoids us having a debate on if two spaces after a period makes sense.